mulesoft/docs-site-ui
# [W-10592995] remove component requirement

ref: [W-10592995](https://gus.lightning.force.com/a07EE00000laaDwYAI)

# Enhancement

- add search bar to the 404 page. This is done by removing page.component from the handlebar logic. Local validation doesn't show any poor behavior on the other pages. 

![Screen Shot 2022-05-04 at 3 50 54 PM](https://user-images.githubusercontent.com/10934908/166838048-b297e8c0-34ea-44a6-a18b-aa0972ae0bd7.png)

# How to test

1. Run `export COVEO_ACCESS_TOKEN=<token_value>`. For now, the value can be found in https://github.com/mulesoft/docs-site-playbook/blob/master/Jenkinsfile#L7
2. Run `gulp preview` for this branch
3. Open http://localhost:5252/404.html in a web browser. Validate that the search bar is present. If it doesn't, make sure the Coveo access token is valid, or else it wouldn't show
4. Click the search bar to open the search modal. Validate that the elements are displaying correctly, which shows that CSS/JS are imported properly. Validate that the searches are returning appropriate results
5. Look at other pages. Validate that the other pages are displaying properly
